test/integration.test.js

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line node/no-unpublished-import
+import { describe, expect, it } from 'vitest';
+import { execaSync } from 'execa';
+
+const inputs = {
+	stylelintConfigExists: 'test/fixtures/stylelint-config-exists',
+};
+
+function getProjectRoot(context) {
+	return context.meta.file.filepath.replace(context.meta.file.name, '');
+}
+
+function setup(pathToTest, projectRoot, args = []) {
+	return execaSync('node', [`${projectRoot}create-stylelint.mjs`, ...args], {
```

Comment:
[suggest] Possible to simplify the code by adding a execute permission to the file:

```console
$ chmod +x create-stylelint.mjs
```

```suggestion
	return execaSync(`${projectRoot}create-stylelint.mjs`, args, {
```

Reply:
It seems like this is necessary for this to work on Windows (I tried the suggestion, and it failed on CI). I'm not familiar with windows development, so perhaps I'm missing something?

Reply:
Ah, sounds good. Let's keep it. Thanks.